ramalama/model.py

```diff
@@ -131,16 +131,14 @@ def _image(self, args):
 
         return f"{image}:latest"
 
-    def setup_container(self, args):
+    def get_container_name(self, args):
```

Comment:
Pleas do not prefix get functions with get_, just call it container_name()

self.container_name()